src/Features/CSharp/Portable/MakeStructFieldsWritable/CSharpMakeStructFieldsWritableCodeFixProvider.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
```

Comment:
needs copyright.

Reply:
Resolved in [https://github.com/dotnet/roslyn/pull/32063/commits/5f00e097f529b82deebe0783a03448f9c954f6ec](https://github.com/dotnet/roslyn/pull/32063/commits/5f00e097f529b82deebe0783a03448f9c954f6ec)